.travis.yml

```diff
@@ -25,7 +38,23 @@ env:
   global:
     - TRIAL_REPORTER=text
 
+# Travis-CI has an old version of PyPy so we install a newer version on the
+# fly.
+# Source: https://github.com/saghul/pycares/blob/master/.travis.yml
 install:
+  - |
+      if [ "$TRAVIS_PYTHON_VERSION" = "pypy" ]; then
+        export PYENV_ROOT="$HOME/.pyenv"
+        if [ -f "$PYENV_ROOT/bin/pyenv" ]; then
+          pushd "$PYENV_ROOT" && git pull && popd
+        else
+          rm -rf "$PYENV_ROOT" && git clone --depth 1 https://github.com/yyuu/pyenv.git "$PYENV_ROOT"
+        fi
+        export PYPY_VERSION="5.3"
+        "$PYENV_ROOT/bin/pyenv" install --skip-existing "pypy-$PYPY_VERSION"
+        virtualenv --python="$PYENV_ROOT/versions/pypy-$PYPY_VERSION/bin/python" "$HOME/virtualenvs/pypy-$PYPY_VERSION"
+        source "$HOME/virtualenvs/pypy-$PYPY_VERSION/bin/activate"
+      fi
   - pip install tox tox-travis
```

Comment:
You don't technically need to install `tox` explicitly, since `tox-travis` depends on `tox`... :-)